.gitignore

```diff
@@ -4,4 +4,7 @@
 /npm-debug.log
 
 # Build output
-/lib
\ No newline at end of file
+/lib
+
+# Test output
+/tmpTest
```

Comment:
Did tests already save output there? Wondering where this camelCased folder name came from :-)